src/rtc_pl031.rs

```diff
@@ -322,6 +334,21 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_regression_year_1970() {
+        // This is a regression test for: https://github.com/rust-vmm/vm-superio/issues/47.
+        // The problem is that the time in the guest would show up as in the 1970s.
+        let mut rtc = RTC::new();
+        let expected_time = get_epoch_time();
+
+        let mut actual_time = [0u8; 4];
+        rtc.read(RTCDR, &mut actual_time);
+        // Check that the difference between the current time, and the time read from the
+        // RTC device is never bigger than one second. This should hold true irrespective of
+        // scheduling.
+        assert!(u32::from_le_bytes(actual_time) - expected_time <= 1);
```

Comment:
If these are integers, should we just check for equality (as it is already done in the other tests)?

Reply:
If my understanding is correct, we cannot check for equality here because there might be 1 second elapsed between RTC::new() and the call on the next line `get_epoch_time`. If this happens, then the test will intermittently fail, and thus is should be safe to assume that that at most 1 second can pass between the two calls.